plugins/module_utils/mh/mixins/cmd.py

```diff
@@ -110,6 +114,15 @@ def custom_formats(self):
             result[param] = ArgFormat(param, **fmt_spec)
         return result
 
+    def __init__(self, *args, **kwargs):
+        super(CmdMixin, self).__init__(*args, **kwargs)
+        self.module.deprecate(
+            'The CmdMixin used in classes CmdModuleHelper and CmdStateModuleHelper is being deprecated. '
+            'Modules should use community.general.plugins.module_utils.cmd_runner.CmdRunner instead.',
+            version='8.0.0',
```

Comment:
In case it turns out someone from outside this collection is relying on it, we should bump this up to `9.0.0`. But as long as nobody is complaining, let's keep 8.0.0 :)